bokehjs/src/coffee/models/graphs/static_layout_provider.coffee

```diff
@@ -18,8 +18,10 @@ export class StaticLayoutProvider extends LayoutProvider
     starts = edge_source.data.start
     ends = edge_source.data.end
     for i in [0...starts.length]
-      start = if @graph_layout[starts[i]]? then @graph_layout[starts[i]] else [null, null]
-      end = if @graph_layout[ends[i]]? then @graph_layout[ends[i]] else [null, null]
+      if @graph_layout[starts[i]]? and @graph_layout[ends[i]]?
```

Comment:
yes. I poorly described the issue originally which made reviewing the PR difficult. sorry about that.